griptape/tasks/prompt_task.py

```diff
@@ -32,6 +33,22 @@ class PromptTask(RuleMixin, BaseTask):
         alias="input",
     )
 
+    @property
+    def rulesets(self) -> list:
+        default_rules = self.rules
+        rulesets = self._rulesets
+
+        if self.structure is not None:
+            if self.structure._rulesets:
+                rulesets = self.structure._rulesets + self._rulesets
```

Comment:
its sort of odd to workaround the logic that is already in the mixin for the `ruleset` property right? can this be refactored at all to be less implentation-dependant? 

seems like we dont want two "default ruleset"s coming from the structure and task but maybe thats ok since they're defined two places?

Reply:
> its sort of odd to workaround the logic that is already in the mixin for the ruleset property right?

I don't find it that odd -- we're overriding a field to add some context specific behavior. I could use `super().rulesets` to cut down on some duplicate code but then we need to figure out how to merge the two.

> seems like we dont want two "default ruleset"s coming from the structure and task but maybe thats ok since they're defined two places?

I don't think we should create two rulesets. A "Default Ruleset" is already ambiguous enough, having a "Default Structure Ruleset" and a "Default Task Ruleset" feels unnecessarily confusing for the LLM.

Reply:
should the default name just be `Rules` or something then?